spec.html

```diff
@@ -28051,8 +28338,24 @@ <h2>Syntax</h2>
 
         ImportedBinding :
           BindingIdentifier[~Yield, +Await]
+
+        WithClause[DeprecatedAssert] :
+          AttributesKeyword[?DeprecatedAssert] `{` `}`
+          AttributesKeyword[?DeprecatedAssert] `{` WithEntries `,`? `}`
+
+        AttributesKeyword[DeprecatedAssert] :
+          [+DeprecatedAssert] `with`
+          [~DeprecatedAssert] `with`
```

Comment:
This is a workaround to suppress a warning about `[DeprecatedAssert]` not being used.